src/test/obj_pmalloc_mt/TEST8

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2023, Intel Corporation
+
+#
+# src/test/obj_pmalloc_mt/TEST8 -- multithreaded allocator test
+#	(long - several iteration) w/ statistics
+#
+
+. ../unittest/unittest.sh
+
+require_valgrind 3.10
+require_fs_type pmem
+require_test_type medium
+configure_valgrind helgrind force-enable
+
+setup
+
+if test -f "rm $DIR/testfile.TEST8"; then
+	rm $DIR/testfile.TEST8
+fi
+PMEM_IS_PMEM_FORCE=1 expect_normal_exit\
+	./obj_pmalloc_mt$EXESUFFIX 32 1000 100 $DIR/testfile.TEST8
+
+if test -f "rm $DIR/testfile.TEST8"; then
```

Comment:
Please assign path to var at the beginning.
You can always delete this file with -f without the if condition. Is there any reason why you check it?

Reply:
Done